AnaProd/tasks.py

```diff
@@ -546,9 +549,16 @@ def run(self):
             AnaTupleFileList_cmd.extend(["--nEventsPerFile", f"{nEventsPerFile}"])
             if sample_name == "data":
                 AnaTupleFileList_cmd.extend(["--isData", "True"])
-                AnaTupleFileList_cmd.extend(
-                    ["--lumi", f'{self.setup.global_params["luminosity"]}']
-                )
+                if self.test:
```

Comment:
Undo this. We want to test that lumi splitting works. 1000 events should never get enough lumi to trigger splitting. If they do, it means the splitting logic must be fixed.